<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Вхід</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../img/img.qrc">
    <normaloff>:/icons/icon.png</normaloff>:/icons/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 69, 71);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Ignored</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>150</width>
        <height>150</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>350</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>350</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(127, 180, 175);
border-radius: 20px;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="studlyLabel">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>84</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Ink Free</family>
               <pointsize>35</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(32, 69, 71);</string>
             </property>
             <property name="text">
              <string>StudLy</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="errorLabel">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="loginLabel">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="lineWidth">
              <number>5</number>
             </property>
             <property name="text">
              <string>Телефон/Email</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
             <property name="margin">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="loginEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777214</height>
              </size>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="tabletTracking">
              <bool>false</bool>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
border-width: 2px;
border-color: rgb(69, 119, 108);
background-color: rgba(69, 119, 108, 125);
border-radius: 5px;</string>
             </property>
             <property name="inputMask">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="passworrdLabel">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>5</number>
             </property>
             <property name="midLineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>Пароль</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
             <property name="margin">
              <number>5</number>
             </property>
             <property name="indent">
              <number>-1</number>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByMouse</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passwordEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777214</height>
              </size>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="tabletTracking">
              <bool>false</bool>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
border-width: 2px;
border-color: rgb(69, 119, 108);
background-color: rgba(69, 119, 108, 125);
border-radius: 5px;</string>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="dragEnabled">
              <bool>false</bool>
             </property>
             <property name="cursorMoveStyle">
              <enum>Qt::LogicalMoveStyle</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
           <property name="spacing">
            <number>7</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QCheckBox" name="show">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);</string>
               </property>
               <property name="text">
                <string>Показати</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="rememberUser">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);</string>
               </property>
               <property name="text">
                <string>Запам'ятати мене</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="loginButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <italic>false</italic>
               <bold>false</bold>
               <underline>false</underline>
               <strikeout>false</strikeout>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
border-style: solid;
border-width: 2px;
border-color: rgb(69, 119, 108);
border-radius: 9px;
background-color: rgba(69, 119, 108, 125);</string>
             </property>
             <property name="text">
              <string>Увійти</string>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="registrationlLabel">
             <property name="font">
              <font>
               <underline>true</underline>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>Не маєте акаунту? Зареєструйтесь.</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Ignored</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>400</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Ignored</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>400</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Ignored</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>150</width>
        <height>150</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../img/img.qrc"/>
 </resources>
 <connections/>
</ui>
